src/Accounts/Accounts/help/Connect-AzAccount.md

```diff
@@ -154,7 +183,25 @@ TenantId         : 4cd76576-b611-43d0-8f2b-adcb139531bf
 Environment      : AzureCloud
 ```
 
-This command connects to an Azure account using certificate-based service principal authentication. Theservice principal used for authentication should have been created with the given certificate.
+This command connects to an Azure account using certificate-based service principal authentication. The service principal used for authentication should have been created with the given certificate.
+
+### Example 8: Add an account using AccessToken authentication
+```powershell
+PS C:\> $url = "https://login.windows.net/<SubscriptionId>/oauth2/token"
+PS C:\> $body = "grant_type=refresh_token&refresh_token=<refreshtoken>" # Refresh token obtained from ~/.azure/TokenCache.dat
+PS C:\> $response = Invoke-RestMethod $url -Method POST -Body $body;
+PS C:\> $AccessToken = $response.access_token;
+PS C:\> $body += "&resource=https%3A%2F%2Fvault.azure.net";
+PS C:\> $response = Invoke-RestMethod $url -Method POST -Body $body;
+PS C:\> $KeyVaultAccessToken = $response.access_token;
+PS C:\> Connect-AzAccount -AccountId "azureuser@contoso.com" -AccessToken $AccessToken -KeyVaultAccessToken $KeyVaultAccessToken -Tenant "xxxx-xxxx-xxxx-xxxx" -SubscriptionId "yyyy-yyyy-yyyy-yyyy"
```

Comment:
I think we want to include a graphAccessToken here as well - many KeyVault cmdlets won't work without this.